vspec/vssexporters/vssidgen.py

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env python3
+#
+# Copyright (c) 2023 Contributors to COVESA
+#
+# This program and the accompanying materials are made available under the
+# terms of the Mozilla Public License 2.0 which is available at
+# https://www.mozilla.org/en-US/MPL/2.0/
+#
+# SPDX-License-Identifier: MPL-2.0
+#
+# Generate IDs of 4bytes size, 3 bytes incremental value + 1 byte for layer id.
+
+import argparse
+import logging
+import os
+from typing import Dict
+from vspec import load_tree
+from vspec.model.constants import VSSTreeType
+from vspec.loggingconfig import initLogging
+from vspec.model.vsstree import VSSNode
+from vspec.utils import vssidval
+import yaml
+
```

Comment:
Add the snippet below. That indicates that this "tool" works with `--no-expand` which it does. (Whether it is useful is a different topic). If you do not think `--no-expand` shall be supported just return false

```

def feature_supported(feature_name: str):
    """Return true for supported arguments/features"""
    if feature_name in ['no_expand']:
        return True
    return False
```